### ISSUE TITLE
Bug in Remote System Discovery - ip neighbour (Ability id: f489321f31b6ef36304294562d3d4645)

### ISSUE THREAD

**Issue by vr-ct** [mitre/caldera#3097]
Running this ability displays the error: sh: 1: Syntax error: ";" unexpected

Cause:  the command is

if [ -x "$(command -v ip)" ]; then : ; else apt-get install iproute2 -y; fi;  ;  ip neighbour show

There is an extra semicolon before 'ip neighbour' show.  Removing this fixes the problem.

**Comment by elegantmoose** [comment]
Correct. This is a known issue. See Atomic readme: https://github.com/mitre/atomic.

We try to not to interfere with any TTPs that get pulled directly from Atomic Red Team corpus.

Local fix: just adjust ability command string on your Caldera instance.

**Comment by deacon-mp** [comment]
This issue has been migrated to the Stockpile plugin repository: mitre/stockpile#596
